pepper-apis/studybuilder-cli/studies/lms/patches/parental-consent-pdf-v2.conf

```diff
@@ -0,0 +1,271 @@
+                        "checkIfFalse": false
+                      },
+                      {
+                        "type": "BOOLEAN",
+                        "field": "tissue_sample_no",
+                        "activityCode": "PARENTAL_CONSENT",
+                        "questionStableId": "PARENTAL_CONSENT_TISSUE",
+                        "checkIfFalse": true
+                      },
+                      {
+                        "type": "TEXT",
+                        "field": "child_first_name",
+                        "activityCode": "PARENTAL_CONSENT",
+                        "questionStableId": "PARENTAL_CONSENT_CHILD_FIRSTNAME"
+                      },
+                      {
+                        "type": "TEXT",
+                        "field": "child_last_name",
+                        "activityCode": "PARENTAL_CONSENT",
+                        "questionStableId": "PARENTAL_CONSENT_CHILD_LASTNAME"
+                      },
+                      {
+                        "type": "TEXT",
+                        "field": "first_name",
+                        "activityCode": "PARENTAL_CONSENT",
+                        "questionStableId": "PARENTAL_CONSENT_FIRSTNAME"
+                      },
+                      {
+                        "type": "TEXT",
+                        "field": "last_name",
+                        "activityCode": "PARENTAL_CONSENT",
+                        "questionStableId": "PARENTAL_CONSENT_LASTNAME"
+                      },
+                      {
+                        "type": "DATE",
+                        "field": "child_date_of_birth",
+                        "activityCode": "PARENTAL_CONSENT",
+                        "questionStableId": "PARENTAL_CONSENT_CHILD_DOB"
+                      },
+                      {
+                        "type": "PICKLIST",
+                        "field": "relationship",
+                        "activityCode": "PARENTAL_CONSENT",
+                        "questionStableId": "PARENTAL_CONSENT_RELATIONSHIP"
+                      },
+                      {
+                        "type": "ACTIVITY_DATE",
+                        "field": "date",
+                        "activityCode": "PARENTAL_CONSENT"
+                      },
+                     {
+                       "type": "TEXT",
+                       "field": "first_name",
+                       "activityCode": "PARENTAL_CONSENT",
+                       "questionStableId": "PARENTAL_CONSENT_FIRSTNAME"
+                     },
+                     {
+                       "type": "TEXT",
+                       "field": "last_name",
+                       "activityCode": "PARENTAL_CONSENT",
+                       "questionStableId": "PARENTAL_CONSENT_LASTNAME"
+                     }
+                    ]
+                  },
+                  {
+                    "filepath": "pdfs/consent_parental_contact_v2.pdf",
```

Comment:
@pegahtah 
Are there any changes to contact pdf template ?

Reply:
yes the proxy names